### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by NEEDcreations** [ZeusLN/zeus#1627] [Ideas]
I would appreciate a 2-Factor-Authentication option to secure our mobile apps.

**Comment by kaloudis** [comment]
I don't understand. You want to use a second form of authentication to get into Zeus? Like enter in your password and then enter in a code from an authenticator app too?

This is a little redundant as the Zeus password is only stored locally. If an attacker somehow gets it they will also be able to get your 2FA key.
